sphinx-doc/module-md-nlist.rst

```diff
@@ -9,12 +9,10 @@ md.nlist
     :nosignatures:
 
     md.nlist.Cell
-    md.nlist.stencil
-    md.nlist.tree
 
 .. rubric:: Details
 
 .. automodule:: hoomd.md.nlist
     :synopsis: Neighbor list acceleration structures.
     :members:
-    :no-inherited-members:
+    :inherited-members:
```

Comment:
```suggestion
    :no-inherited-members:
    :show-inheritance:
```
We are standardizing HOOMD docs to use `no-inerherited-members` and only document comment methods once in the base class. `show-inheritance` provides a link to that base class.